src/main/java/se/sundsvall/comfactfacade/api/SigningResource.java

```diff
@@ -39,14 +39,16 @@
 @Tag(name = "Signings", description = "Signing operations")
 @ApiResponse(responseCode = "400", description = "Bad request", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(oneOf = {Problem.class, ConstraintViolationProblem.class})))
 @ApiResponse(responseCode = "500", description = "Internal Server error", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
+@ApiResponse(responseCode = "502", description = "Bad Gateway", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
 public class SigningResource {
 
 	private final SigningService signingService;
 
 	public SigningResource(final SigningService signingService) {this.signingService = signingService;}
 
 	@GetMapping(produces = {APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE})
-	@Operation(summary = "Get all signing instances.")
+	@Operation(summary = "Get all signing instances.", description = "The 'sort' parameter in the Pageable object can take the values 'SigningInstanceId', 'ReferenceNumber', 'CustomerReferenceNumber', 'Created', 'Changed', 'Expires', 'StatusCode', 'StatusMessage', 'UserId', 'Language', 'SignatoryReminderStartDate', 'InitiatorEmailAddress', 'QueueCreated', 'QueueChanged'.")
```

Comment:
sort parameter is now mandatory, maybe make it optional and default it with 'SigningInstanceId' if not provided?